test/test_torch.py

```diff
@@ -8200,6 +8200,18 @@ def test_expected_failure_xla(self, device):
         if self.device_type == 'xla':
             self.assertTrue(False)
 
+    def test_assertRaisesRegex_ignore_msg_non_native_device(self, device):
```

Comment:
Let's add a comment explaining this test is here because PyTorch/XLA runs test_torch.py as parts of its CI (as opposed to being in test_testing.py, for example)